kafka-base/image.yaml

```diff
@@ -24,6 +26,8 @@ envs:
     value: "2.0.0"
```

Comment:
is this right? or it should be 2.1.0? @scholzj ?

Reply:
Well, if it is for the 2.0.0 image it should be 2.é.0. If it is for the 2.1.1 image it should be 2.1.1. If it is for both we should probably put there 2.1.1.

Reply:
This Image descriptor file will act as a template for the 2.0.0 and 2.1.1 versioned image descriptor files that I will generate and push later. The generator will populate version specific fields like these with the appropriate versions. However I was planning on removing this env var later in a refactor as its only used for upstream build scripts [1].

Regardless, I'll blank out the these fields when I add the version specific image descriptor files to make it clear that this is to be used as a template file

[1] https://github.com/jboss-openshift/cct_module/tree/jmtd-patch-1bmit to I'm planning on taking this env var out https://github.com/jboss-container-images/amqstreams-1-openshift-image/issues/72

Reply:
Sounds good.